### PR TITLE
Special Breakpoints Dropdown

### PR DESCRIPTION
**Pull request by GarethSharpe**
Associated Issue: #4109

### CODE REVIEW

File: src/actions/pause.js [hunk 1]
```diff
@@ -141,12 +142,15 @@ export function pauseOnExceptions(
  */
 export function command(type: CommandType) {
   return async ({ dispatch, client }: ThunkArgs) => {
-    // execute debugger thread command e.g. stepIn, stepOver
-    dispatch({ type: "COMMAND", value: { type } });
-
-    await client[type]();
-
-    dispatch({ type: "CLEAR_COMMAND" });
+    dispatch({
```
Comment (jasonLaster) [inline src/actions/pause.js]: you don't need to do this refactor. That was me messing around

File: src/reducers/pause.js [hunk 2]
```diff
@@ -141,10 +141,10 @@ function update(state: PauseState = State(), action: Action): PauseState {
       });
 
     case "COMMAND":
-      return { ...state, command: action.value.type };
-
-    case "CLEAR_COMMAND":
-      return { ...state, command: "" };
+      return {
```
Comment (jasonLaster) [inline src/reducers/pause.js]: you don't need this

**Comment by jasonLaster** [comment]
Hey! I just took a look and it seems like you pulled in my previous PR pretty literally...

So, this is a tempting thing to do... but I actually think it gets you in more trouble than it's worth:

1. the changes are out of date
2. the PR has tons of stuff that's not really needed
3. the PR is big and hard to understand... really hard
4. it's not clear what to do (obviously)

Here's what I recommend.

1. start on a new up to date branch
2. add a dropdown to the the breakpoints header (look at the PR to see how i did it). Maybe make a small PR for customizing the dropdown button :) Maybe make a small PR that adds this dropdown behind a feature [flag][f]
3. add one action to the dropdown (perhaps pause on exceptions)... make sure that works. Add all the other actions...
4. start removing the buttons from the command bar
5. profit

**Comment by GarethSharpe** [comment]
Hey man - Ha yes I did. I was under the (false it seems) impression that
everything that you did in that PR was required for the functionality so I
just started trying to understand how it all worked. Thank you very much
for your recommendations though. I really like that approach and I think it
will give you a little more to go off of if I ever message you again and
say "I'm stuck". Thanks again for putting in the time - I'll start this
Saturday morning. #profit

**Pull request closed without merging**